scrapi/harvesters/biomedcentral.py

```diff
@@ -61,7 +65,8 @@ def schema(self):
             'contributors': ('/authorNames', process_contributors),
             'uris': {
                 'canonicalUri': '/articleFullUrl',
-                'objectUris': ('/doi', lambda x: ['http://dx.doi.org/' + x])
+                'objectUris': ('/doi', lambda x: ['http://dx.doi.org/' + x]),
+                'providerUris': ('/articleFullUrl', '/abstractPath', join_uris)
             },
             'title': ('/bibliographyTitle', '/blurbTitle', lambda x, y: x or y),
             'providerUpdatedDateTime': ('/published Date', lambda x: date_formatter(x)),
```

Comment:
realize this was probably me, but you can just say date_formatter instead of lambda x: date_formatter(x)